ec_ms_quantification/EC-MS_ixdat_tutorial.ipynb

```diff
@@ -0,0 +1,1017 @@
+    "print(cal_O2_2) \n",
+    "\n",
+    "#ethylene\n",
+    "meas_gascal_C2H4 = Measurement.read(r\"./2022-07-20 13_04_48 gas_cal_C2H4.tsv\", reader=\"zilien\", \n",
+    "                                 technique=\"MS\")\n",
+    "\n",
+    "axes_i = meas_gascal_C2H4.plot_measurement()\n",
+    "\n",
+    "cal_C2H4, ax_j = chip.gas_flux_calibration_curve(\n",
+    "    measurement=meas_gascal_C2H4,  # the measurement with the calibration data\n",
+    "    mol=\"C2H4\",  # the molecule to calibrate\n",
+    "    mass=\"M26\",  # the mass to calibrate at\n",
+    "    tspan_list=[\n",
+    "        [3200, 3400],\n",
+    "        [4400, 4600],\n",
+    "        [5800, 6000],\n",
+    "        [6900, 7100],\n",
+    "        [8000, 8200],\n",
+    "         ], # the timespans to average the signal over\n",
+    "    carrier_mol=\"He\", # the molecule that is used to calculate the total flux through the capillary\n",
+    "    mol_conc_ppm= [5000, 10000, 20000, 50000, 100000], # the concentration of the analyte at the timespans given above in ppm\n",
+    "    tspan_bg=[900, 1000], # the timespan for background subtraction (optional)\n",
+    "    return_ax=True, # whether to return the calibration curve\n",
+    "    #axes_measurement=axes_i # the axis on which to indicate what signal is used with a thicker line  \n",
+    "    #-> this introduces an error into the data analysis (defines that mol_conc_ppm = 10^6) FIXME!! \n",
+    ")\n",
+    "print(cal_C2H4)\n",
+    "\n",
+    "# add these gas calibrations to the list:\n",
+    "cal_results.append(cal_O2_2)\n",
+    "cal_results.append(cal_C2H4)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "843aa107",
+   "metadata": {},
+   "source": [
+    "Saving the calibration\n",
+    "-------------------------------\n",
+    "\n",
+    "We might want to use the calibration for other measurements later, so let's create an ``ECMSCalibration`` object and save the calibration."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ca2699d4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ixdat.techniques.ec_ms import ECMSCalibration\n",
+    "\n",
+    "# Now we put all these results together in an ECMSCalibration object:\n",
+    "calibration = ECMSCalibration(\n",
+    "    name=\"SI_quant_appnote_calibration\",  # Named for the article\n",
+    "    date=\"22-07-20\",  # date of the calibration measurements\n",
+    "    setup=\"SI R&D1\",  # setup where calibration was made\n",
+    "    ms_cal_results=cal_results,  # the mass spec calibrations\n",
+    "    RE_vs_RHE=0,  # the RE potential in [V]\n",
+    "    A_el=0.197,  # the geometric electrode area in [cm^2]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9a8ec94c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# save the calibration:\n",
```

Comment:
This could be its own markdown and code blocks. You could mention that the calibration is human-readable in notepad.